components/brave_wallet_ui/page/screens/composer_ui/token_list_item/token_list_item.tsx

```diff
@@ -67,151 +74,212 @@ export const TokenListItem = React.forwardRef<HTMLDivElement, Props>(
+                      <DisabledLabel>{getLocale(disabledText)}</DisabledLabel>
+                    </>
+                  )}
+                </Row>
                 <NetworkAndFiatText
                   textSize='12px'
                   isBold={false}
-                  textAlign='right'
+                  textAlign='left'
                 >
-                  {formattedFiatBalance}
+                  {formatTokenBalanceWithSymbol(
+                    token.assetBalance,
+                    token.asset.decimals,
+                    token.asset.symbol
+                  )}
                 </NetworkAndFiatText>
+              </Column>
+            </IconAndName>
+            {!token.asset.isErc721 &&
+              !token.asset.isNft &&
+              !token.asset.isErc1155 &&
+              tokenHasBalance && (
+                <Column alignItems='flex-end'>
+                  {isLoadingSpotPrice ? (
+                    <>
+                      <LoadingSkeleton
+                        width={60}
+                        height={14}
+                      />
+                      <VerticalSpace space='4px' />
+                      <LoadingSkeleton
+                        width={60}
+                        height={12}
+                      />
+                    </>
+                  ) : (
+                    <>
+                      <Row width='unset'>
+                        {tokenHasMultipleAccounts && <AccountsIcon />}
+                        <NameAndBalanceText
+                          textSize='14px'
+                          isBold={true}
+                          textAlign='right'
+                        >
+                          {formattedFiatBalance}
+                        </NameAndBalanceText>
+                      </Row>
+                      <Row width='unset'>
+                        <PercentChangeIcon
+                          name={
+                            isPriceDown
+                              ? 'arrow-diagonal-down-right'
+                              : 'arrow-diagonal-up-right'
+                          }
+                          isDown={isPriceDown}
+                        />
+                        <PercentChangeText
+                          textSize='12px'
+                          isBold={false}
+                          textAlign='right'
+                          isDown={isPriceDown}
+                        >
+                          {`${Math.abs(
+                            Number(spotPrice?.assetTimeframeChange ?? '')
+                          ).toFixed(2)}%`}
+                        </PercentChangeText>
+                      </Row>
+                    </>
+                  )}
+                </Column>
               )}
-            </Column>
+          </Button>
+          {(!tokenHasBalance ||
+            token.asset.isErc721 ||
+            token.asset.isNft ||
+            token.asset.isErc1155) && (
```

Comment:
perf-nit: might run faster if we check `isNft` first

Reply:
Moved!